packages/apps/reputation-oracle/server/src/modules/user/user.dto.ts

```diff
@@ -50,6 +51,30 @@ export class UserUpdateDto {
   public status?: UserStatus;
 }
 
+export class RegisterLabelerRequestDto {
+  @ApiProperty({ name: 'chain_id' })
+  @IsEnum(ChainId)
+  public chainId: ChainId;
+
+  @ApiProperty()
+  @IsEthereumAddress()
+  public address: string;
+
+  @ApiProperty({ name: 'type' })
+  @IsEnum(OracleType)
+  public type: OracleType;
+
+  @ApiProperty()
+  @IsString()
+  public country: string;
+}
+
+export class RegisterLabelerResponseDto {
+  @ApiProperty({ name: 'signed_site_key' })
+  @IsString()
+  public signedSiteKey: string;
```

Comment:
Since we won't store hcaptcha sitekey in kvstore, we can return a plain sitekey.